### PR TITLE
Create script-Append.yml

### PR DESCRIPTION
**Pull request by meirwah**
add append script

### CODE REVIEW

File: Scripts/script-Append.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  res = {
+      'ContentsFormat': formats['text'],
+      'Type': entryTypes['note'],
+      'Contents': result,
+      'ReadableContentsFormat': formats['text'],
+      'HumanReadable': result,
+      'EntryContext': {demisto.args()['key']: result}
+  }
+  demisto.results(res)
+type: python
```
Comment (dorsha) [inline Scripts/script-Append.yml]: Because is performance I would do this in JavaScript

File: Scripts/script-Append.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+name: Append
+script: |
+  result = demisto.args()["prefix"] + demisto.args()["suffix"]
+  res = {
+      'ContentsFormat': formats['text'],
+      'Type': entryTypes['note'],
+      'Contents': result,
+      'ReadableContentsFormat': formats['text'],
+      'HumanReadable': result,
+      'EntryContext': {demisto.args()['key']: result}
```
Comment (dorsha) [inline Scripts/script-Append.yml]: Should be with output and not context.

File: Scripts/script-Append.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  version: -1
+name: Append
+script: |
+  result = demisto.args()["prefix"] + demisto.args()["suffix"]
+  res = {
+      'ContentsFormat': formats['text'],
+      'Type': entryTypes['note'],
+      'Contents': result,
+      'ReadableContentsFormat': formats['text'],
+      'HumanReadable': result,
```
Comment (dorsha) [inline Scripts/script-Append.yml]: I would not add entry in the war room only for append strings, too noisy and not relevant for the investigation.
Should have output only

Comment (meirwah) [inline Scripts/script-Append.yml]: you have any example for python script with output?

File: Scripts/script-Append.yml (hunk 2 above)
Comment (meirwah) [inline Scripts/script-Append.yml]: you have any example for python script with output?

File: Scripts/script-Append.yml (hunk 1 above)
Comment (meirwah) [inline Scripts/script-Append.yml]: disagree on that, I think readability is more important here

Comment (dorsha) [inline Scripts/script-Append.yml]: If it is that important to add garbage to the war room, please provide better human readable  message to the war room, no one will understand that.

Comment (dorsha) [inline Scripts/script-Append.yml]: And what how war room readability related to performance?

File: Scripts/script-Append.yml (hunk 2 above)
Comment (dorsha) [inline Scripts/script-Append.yml]: The same as in JavaScript

File: Scripts/script-Append.yml (hunk 1 above)
Comment (meirwah) [inline Scripts/script-Append.yml]: code readability

File: Scripts/script-Append.yml (hunk 2 above)
Comment (dorsha) [inline Scripts/script-Append.yml]: script-UnzipFile.yml and there are many other python examples in the content repo

**Pull request closed without merging**